Gems/EMotionFX/Code/EMotionFX/Rendering/OpenGL2/Source/GLSLShader.cpp

```diff
@@ -263,8 +250,8 @@ namespace RenderGL
     // FindAttribute
     GLSLShader::ShaderParameter* GLSLShader::FindAttribute(const char* name)
     {
-        const uint32 index = FindAttributeIndex(name);
-        if (index == MCORE_INVALIDINDEX32)
+        const size_t index = FindAttributeIndex(name);
+        if (index == InvalidIndex)
```

Comment:
Is this automatically dereferenced to the correct type? InvalidaIndex32/64 etc? I am a bit scared about that one as I remember we had some cases of using a size_t while still using the InvalidIndex32 - which is bad of course, but this might break?

Reply:
`InvalidIndex` is the invalid index for `size_t`:
```cpp
constexpr size_t InvalidIndex = 0xffffffffffffffff;
```
So it will be the right value for whatever size `size_t` is on the current platform.